src/sqsd.groovy

```diff
@@ -119,5 +119,5 @@ def handleMessage(String httpHost, String httpPath, String contentType, Message
 
     println "POST " + httpHost + httpPath + " :: " + status
 
-    status < 400
+    status == 200
```

Comment:
If it were for me, the valid status codes would have been 200 and 202. However, as per SQS documentation, as @tkleinhakisa pointed out, the only valid status is 200. Will proceed with pull.